packages/core/src/__integrationtests__/Attestation.spec.ts

```diff
@@ -173,7 +173,9 @@ describe('When there is an attester, claimer and ctype drivers license', () => {
       tokenHolder.address
     )
     await submitExtrinsic(authorizedStoreTx, tokenHolder)
-    expect(await Attestation.checkValidity(attestation.claimHash)).toBe(true)
+    const storedAttestation = await Attestation.query(attestation.claimHash)
+    expect(storedAttestation).not.toBeNull()
```

Comment:
Do you think we need this double check? Would the next line fail when undefined !== false?

Reply:
agreed, seems like `(await Attestation.query(attestation.claimHash)?.revoked === false)` could be our recommended check to see whether an attestation is valid

Reply:
yeah, was thinking the same. Just added it for completeness.